_posts/2024-01-24-trust-in-open-source-software.md

```diff
@@ -22,14 +22,14 @@ While transparency is a key pillar of open-source projects, questions regarding
+We hosted the survey on [OpenSearch.org](http://opensearch.org/) in Q2 and Q3 of 2023 and publicized it through partner communications and community meetings. There were 36 community members who participated in the survey. 91.7% of the sample indicated that they used OpenSearch in a self-service capacity, 66.7% of the sample identified as belonging to the Infra role, and 63.9% of the sample used OpenSearch to power their log analytics use cases. This sample expressed a relatively positive perception of OpenSearch, with an NPS of 64 (as compared to 57.8 in Q1). The users who took the survey represented a wide variety of organizations, with 22% part of large organizations with over 10,000 employees.
 
 <img width=350 src="/assets/media/blog-images/2024-01-24-trust-in-open-source-software/Trust Presentation Graphs-01.png">
 <img width=350 src="/assets/media/blog-images/2024-01-24-trust-in-open-source-software/Trust Presentation Graphs-02.png">
 <img width=350 src="/assets/media/blog-images/2024-01-24-trust-in-open-source-software/Trust Presentation Graphs-03.png">
 <img width=350 src="/assets/media/blog-images/2024-01-24-trust-in-open-source-software/Trust Presentation Graphs-04.png">
 
-The key measure we were interested in capturing was on Trust in OpenSearch. The sample indicated a 7.4 on 10 average score on Trust. This measure comprised of the 6 measures. We used K D Paine & Partners 6 factor scale. Aspects of this measure included perceived dependability (4 item; 7.8; alpha =.89), perceived sense of control mutuality (3 item measure; 7.4; alpha = .96), perceived commitment (5 item measure; 7.5; alpha = .94), satisfaction, (5 item measure; 7.1, alpha = .95) communal relationship quality (2 item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5).  Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development). 
+We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure comprised the 6 measures. We used the K D Paine & Partners 6-factor scale. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development). 
```

Comment:
It's not clear what we mean by "This measure comprised the 6 measures", and the phrase is redundant. Which measures? If we mean the ones referenced in the following sentence, we can just delete this sentence and say "We used the K D Paine & Partners 6-factor scale for this survey." However, I can't find any real references to this organization/scale online. Is the naming of the organization and the scale correct as written?

Reply:
This measure comprised of 6 dimensions

Reply:
https://www.instituteforpr.org/wp-content/uploads/2003_MeasuringTrust.pdf

Reply:
@apasun I have made the necessary changes based on your feedback. Thanks!